mint.json

```diff
@@ -53,8 +53,14 @@
         "introduction",
         "concepts",
         "installation",
-        "integration",
-        "architecture"
+        "integration"
+      ]
+    },
+    {
+      "group": "Operations",
```

Comment:
wdyt about putting this section at the bottom, so the heirarcy would be:

- Home
- Configuration
- Authentication
- Operations

Reply:
Yeah, sounds great.